parser/importer_pytesseract.py

```diff
@@ -0,0 +1,170 @@
+# Python3
+# coding: utf-8
+#
+# Commit note of Daniela Grothe
+# This version uses pytesseract.
+# Only tested on Windows 8 with Python 3.8
+# and ImageMagick 6.8.8-10 Q8 x64
+# Seems as if either Win8-cmd or ImageMagick didn't want 
+# as many high-commas (didn't find my files, error msg such as:
+# "convert.exe: unable to open image `\Users\daniela\..."
+# ) so I removed some of them in the convert commands.
+# 
+#
+# Copyright 2015-2018
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+
+from PIL import Image
+
+BASE_PATH = os.getcwd()
+INPUT_FOLDER = os.path.join(BASE_PATH, "data\\img")
+TMP_FOLDER = os.path.join(BASE_PATH, "data\\tmp")
+OUTPUT_FOLDER = os.path.join(BASE_PATH, "data\\txt")
+
+# --- pytesseract prep
+try:
+    from PIL import Image
+except ImportError:
+    import Image
+import pytesseract
+
+pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract.exe'
+the_chosen_lang = 'deu'
+
+# --- End of pytesseract prep
+
+def prepare_folders():
+    """
+    :return: void
+        Creates necessary folders
+    """
+
+    for folder in [
+        INPUT_FOLDER, TMP_FOLDER, OUTPUT_FOLDER
+    ]:
+        if not os.path.exists(folder):
+            os.makedirs(folder)
+
+
+def find_images(folder):
+    """
+    :param folder: str
+        Path to folder to search
+    :return: generator of str
+        List of images in folder
+    """
+
+    for file in os.listdir(folder):
+        full_path = os.path.join(folder, file)
+        if os.path.isfile(full_path):
+            try:
+                _ = Image.open(full_path)  # if constructor succeeds
+                yield file
+            except:
+                pass
+
+
+def rotate_image(input_file, output_file, angle=90):
+    """
+    :param input_file: str
+        Path to image to rotate
+    :param output_file: str
+        Path to output image
+    :param angle: float
+        Angle to rotate
+    :return: void
+        Rotates image and saves result
+    """
+    # 
+    cmd = "convert" + " -rotate " + str(angle)
+    # errors on Windows:
+    # cmd += " '" + input_file + "' '" + output_file + "'"
```

Comment:
That's interesting! I wonder why the old command fails.
If you have some time, you could print the old and the new command on the commandline and see if there is a difference. Maybe it's because of globbing (a `*` somewhere in the command)?